src/test/xml_coverage.go

```diff
@@ -61,3 +159,55 @@ type xmlCoverageLine struct {
 	Hits   int `xml:"hits,attr"`
 	Number int `xml:"number,attr"`
 }
+
+// Coverage struct for writing to xml file
```

Comment:
This (and following types) should not be exported, since they are only for serialisation